### ISSUE TITLE
Logging Metrics using MLFLowlogger to display on MLFlow UI

### ISSUE THREAD

**Issue by jitani** [williamFalcon/pytorch-lightning#576]
#### What is your question?    
I am trying to implement the mlflow logger into my code to log my metrics (train_loss, val loss) and for some reason my code is not working. I was able to log the hyperparams, yet still having trouble with the metrics. Also the run id stored for the mlruns is different that the run id generated by the logger. 

#### Code    

        tqdm_dict = {'train_loss': loss.item() or 0.0}
        output = OrderedDict({
            'loss': loss,
            'progress_bar': tqdm_dict,
            'log': tqdm_dict
        })

               
        return output
#### What have you tried?    
#print(inspect.getargspec(self.logger.experiment.set_tag))
        print(mlflow.start_run(run_name=self.logger.run_uuid, nested=True))
        print(self.logger.experiment.set_tag(run_id= self.logger._run_id, key='train_loss', value=loss))
        #run__id = self.logger.experiment.get_run(self.logger._run_id)
        #for key, value in vars(output):
         #   self.logger.experiment.log_metric(key, value)
        #self.logger.log_metrics(output.items()) #generates tensor error 

in addition to trying to get the same run ID generated by the mlrun.
print(self.logger._run_id)
print(self.logger.run_id)
both generate the same run id which is not what I am looking for 

#### What's your environment?   
Virtual Box on windows 10: Ubuntu 18.04 LTS
- conda version (no venv)   
conda 4.7.12
- PyTorch version   
1.0.2
- Lightning version   
0.5.3.2
- Test-tube version

**Comment by williamFalcon** [comment]
@neggert

**Comment by neggert** [comment]
I'm not clear on what the exact problem you're facing is. Could you post a more complete example, along with the error you're getting?

For starters, though, PyTorch 1.0.2 isn't supported. You need at least 1.1.

You don't need to do `mlflow.start_run` manually. Lightning takes care of this for you if you use the [MLFlowLogger](https://williamfalcon.github.io/pytorch-lightning/Trainer/Logging/#mlflow).

**Comment by jitani** [comment]
> I'm not clear on what the exact problem you're facing is. Could you post a more complete example, along with the error you're getting?
> 
> For starters, though, PyTorch 1.0.2 isn't supported. You need at least 1.1.
> 
> You don't need to do `mlflow.start_run` manually. Lightning takes care of this for you if you use the [MLFlowLogger](https://williamfalcon.github.io/pytorch-lightning/Trainer/Logging/#mlflow).

I updated my PyTorch library to 1.1. Furthermore, I removed the 'mlflow.start_run' and removed the tracking URI arg from the MLFlowLogger(...) and it seemed to do the trick. 

I do have another issue if you could help me with that as well. Regarding the checkpoints. Shall I ask it here or a new thread? It involves the checkpint_callback creating a 'None' folder. 

I apologize in advance, this is my first post on GitHub

**Comment by neggert** [comment]
Pleas start a new issue or stop by our Slack channel (link in readme)

Sent from my iPhone

On Dec 5, 2019, at 3:47 AM, jitani <notifications@github.com> wrote:

﻿

I'm not clear on what the exact problem you're facing is. Could you post a more complete example, along with the error you're getting?

For starters, though, PyTorch 1.0.2 isn't supported. You need at least 1.1.

You don't need to do mlflow.start_run manually. Lightning takes care of this for you if you use the MLFlowLogger<https://williamfalcon.github.io/pytorch-lightning/Trainer/Logging/#mlflow>.

I updated my PyTorch library to 1.1. Furthermore, I removed the 'mlflow.start_run' and removed the tracking URI arg from the MLFlowLogger(...) and it seemed to do the trick.

I do have another issue if you could help me with that as well. Regarding the checkpoints. Shall I ask it here or a new thread? It involves the checkpint_callback creating a 'None' folder.

I apologize in advance, this is my first post on GitHub
